tests/test_upload_mission.py

```diff
@@ -23,15 +33,18 @@ def test_upload_edits_files(base_config: Path, mission_dir: Path):
         base_config_file=base_config, target_folder=mission_dir
     )
     # When Zeus calls "/zeus-upload"
-    modlist = extract_mods(MODLIST_JSON)
+    modlist = extract_mods(MODLIST_JSON, strip_versions)
     out_path = config_gen.zeus_upload(scenario_id, filename, modlist)
     # Then a new server config file is created
     assert out_path.is_file(), "Should have generated a file on disk"
     # And the config file is patched with <modlist.json> and <scenarioId>
     config = json.loads(out_path.read_text())
     assert config["game"]["scenarioId"] == scenario_id, "Should update scenarioId"
     assert isinstance(config["game"]["mods"], list)
-    assert config["game"]["mods"][0] == MODLIST_DICT[0]
+    mods = MODLIST_DICT
+    if strip_versions:
+        mods = MODLIST_DICT_VERSIONLESS
```

Comment:
```suggestion
    mods = MODLIST_DICT if strip_versions else MODLIST_DICT_VERSIONLESS
```

OR EVEN BETTER, see above for making it into the fixture